src/Data/Maybe/Properties.agda

```diff
@@ -65,6 +65,13 @@ map-cong₂ : ∀ {f g : A → B} {mx} →
 map-cong₂ (just eq) = cong just eq
 map-cong₂ nothing   = refl
 
+map-injective : ∀ {f : A → B} {mx my : Maybe A}
```

Comment:
The type can be written as `∀ {f : A → B} → Injective f _≡_ _≡_ → Injective (map f) _≡_ _≡_`